Source/UETorch/Private/UETorch.cpp

```diff
@@ -118,6 +118,10 @@ typedef struct {
  */
 extern "C" void GetViewportSize(IntSize* r)
 {
+  if(GEngine->GameViewport->Viewport == NULL){
```

Comment:
According to [this](https://docs.unrealengine.com/latest/INT/Programming/Slate/InGameUI/index.html), both GEngine and GameViewport can be NULL, I'll the checks for both of them.